apps/qubit/modules/actor/actions/browseAction.class.php

```diff
@@ -81,6 +81,19 @@ public function execute($request)
   {
     parent::execute($request);
 
+    if (isset($request->repos) && ctype_digit($request->repos))
+    {
+      $this->repos = QubitRepository::getById($request->repos);
+
+      // Add repo to the user session as realm
+      $this->context->user->setAttribute('search-realm', $request->repos);
+    }
+    elseif (sfConfig::get('app_enable_institutional_scoping'))
```

Comment:
It's a nitpick but we use 'else if' when outside of templates & using the weird alternative syntax. 

elseif as all one word is only ever required in the <?php elseif (...): ?> style syntax.